tensorflow/c/c_api_experimental.cc

```diff
@@ -46,6 +50,8 @@ limitations under the License.
 #include "tensorflow/core/platform/strcat.h"
 #include "tensorflow/core/protobuf/config.pb.h"
 #include "tensorflow/core/protobuf/tensorflow_server.pb.h"
+#include "tensorflow/core/lib/core/refcount.h"
```

Comment:
This file just includes `tensorflow/core/platform/refcount.h`. Let's just include the original file directly.
```suggestion
#include "tensorflow/core/platform/refcount.h"
```

Reply:
Done.